### PR TITLE
Added patchendpoint for listings

### PR DESCRIPTION
**Pull request by MarkusWallnerFul**

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -1482,6 +1541,24 @@ definitions:
...
+        format: "int64"
+        example: "42"
+        description: "The version of the document to be used in optimistic locking mechanisms."
+      actions:
+        type: "array"
+        minItems: 1
+        items:
+          $ref: '#/definitions/AbstractModificationAction'
+    xml:
+      name: "FacilityPatchActions"
```
Comment (florian-bartels) [inline api.swagger.yaml:1561]: no facilitypatch here

Comment (MarkusWallnerFul) [reply api.swagger.yaml]: corrected it

File: api.swagger.yaml [hunk 2]
```diff
@@ -400,6 +400,65 @@ paths:
...
+          type: "string"
+        - in: "body"
+          name: "body"
+          description: "listings for put"
+          required: true
+          schema:
+            $ref: "#/definitions/ListingPatchActions"
+      responses:
+        200:
+          description: "Facility listing was foundand successfully patched"
```
Comment (evilass) [inline api.swagger.yaml:436]:
```suggestion
          description: "Facility listing was found and successfully patched"
```

File: api.swagger.yaml [hunk 3]
```diff
@@ -1514,7 +1592,41 @@ definitions:
...
+          subtitle:
+            type: "string"
+            example: "44 2/3"
+            description: "a subtitle describing the article"
+          imageUrl:
+            type: "string"
+            description: "A web link to a picture of this article. Please make sure that no authentication is required to fetch the image!"
+        additionalProperties: false
+    xml:
+      name: "ModifyPickJobAction"
```
Comment (evilass) [inline api.swagger.yaml:1629]:
```suggestion
      name: "ModifyListingAction"
```

**Pull request merged**